2021-06-05 09:12:06,581 INFO    MainThread:5916 [wandb_setup.py:_flush():69] setting env: {}
2021-06-05 09:12:06,581 INFO    MainThread:5916 [wandb_setup.py:_flush():69] setting login settings: {'anonymous': 'allow'}
2021-06-05 09:12:06,581 INFO    MainThread:5916 [wandb_init.py:_log_setup():336] Logging user logs to C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210605_091206-15qo76xw\logs\debug.log
2021-06-05 09:12:06,581 INFO    MainThread:5916 [wandb_init.py:_log_setup():337] Logging internal logs to C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210605_091206-15qo76xw\logs\debug-internal.log
2021-06-05 09:12:06,581 INFO    MainThread:5916 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000205EB5E6A88>
2021-06-05 09:12:06,583 INFO    MainThread:5916 [wandb_init.py:init():369] calling init triggers
2021-06-05 09:12:06,583 INFO    MainThread:5916 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-06-05 09:12:06,583 INFO    MainThread:5916 [wandb_init.py:init():418] starting backend
2021-06-05 09:12:06,583 INFO    MainThread:5916 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-06-05 09:12:06,591 INFO    MainThread:5916 [backend.py:ensure_launched():132] starting backend process...
2021-06-05 09:12:06,650 INFO    MainThread:5916 [backend.py:ensure_launched():137] started backend process with pid: 25452
2021-06-05 09:12:06,651 INFO    MainThread:5916 [wandb_init.py:init():423] backend started and connected
2021-06-05 09:12:06,651 INFO    MainThread:5916 [wandb_init.py:init():465] updated telemetry
2021-06-05 09:12:06,652 INFO    MainThread:5916 [wandb_init.py:init():484] communicating current version
2021-06-05 09:12:07,523 INFO    MainThread:5916 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-06-05 09:12:07,524 INFO    MainThread:5916 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-06-05 09:12:07,759 INFO    MainThread:5916 [wandb_init.py:init():522] starting run threads in backend
2021-06-05 09:12:12,008 INFO    MainThread:5916 [wandb_run.py:_console_start():1565] atexit reg
2021-06-05 09:12:12,009 INFO    MainThread:5916 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2021-06-05 09:12:12,009 INFO    MainThread:5916 [wandb_run.py:_redirect():1455] Wrapping output streams.
2021-06-05 09:12:12,009 INFO    MainThread:5916 [wandb_run.py:_redirect():1479] Redirects installed.
2021-06-05 09:12:12,009 INFO    MainThread:5916 [wandb_init.py:init():546] run started, returning control to user process
2021-06-05 09:12:14,323 INFO    MainThread:5916 [wandb_init.py:_pause_backend():249] pausing backend
2021-06-05 09:12:14,323 INFO    MainThread:5916 [jupyter.py:save_ipynb():265] not saving jupyter notebook
